README.md

```diff
@@ -57,31 +57,54 @@ Before you can use the AVM Debugger extension, you need to ensure that you have
 
 In order to use the AVM Debugger extension, you need:
 
-1. TEAL Source Maps. A `*.trace.avm.json` file that maps the compiled TEAL source maps to the original source code. See an example [here](./examples/multiRootWorkspace/slot-machine/debug_traces/simulate-response.trace.avm.json).
-2. Simulate Traces. A `sources.avm.json` file that contains the traces obtained from algod's [`simulate` endpoint](https://developer.algorand.org/docs/get-details/dapps/smart-contracts/debugging/?from_query=simulate#simulate). This serves as an entry point for the debugger. See an example [here](./examples/multiRootWorkspace/slot-machine/.algokit/sources/sources.avm.json).
+1. Simulate Traces. A `*.trace.avm.json` file that maps the compiled TEAL source maps to the original source code. See an example [here](./examples/multiRootWorkspace/slot-machine/debug_traces/simulate-response.trace.avm.json).
+2. TEAL Source Maps. A `sources.avm.json` file that contains the traces obtained from algod's [`simulate` endpoint](https://developer.algorand.org/docs/get-details/dapps/smart-contracts/debugging/?from_query=simulate#simulate). This serves as an entry point for the debugger. See an example [here](./examples/multiRootWorkspace/slot-machine/.algokit/sources/sources.avm.json).
```

Comment:
I'm wondering if we should even document this, given that we dynamically build it rather than requiring it to launch.

Reply:
I think just the descriptions should we reworked. The first part of each point is correct, but the description for each is incorrect

e.g.

```
1. Simulate Traces. A `*.trace.avm.json`  file that contains the traces obtained from algod's [`simulate` endpoint](https://developer.algorand.org/docs/get-details/dapps/smart-contracts/debugging/?from_query=simulate#simulate). This serves as an entry point for the debugger. See an example [here](./examples/multiRootWorkspace/slot-machine/debug_traces/simulate-response.trace.avm.json).
2. Source Maps. Source maps for the programs executed within the trace. These can be either a 
    * a TEAL source map, obtained from algod when compiling a TEAL program
    * a Puya source map, produced when compiling a Smart Contract with the Puya compiler
```

Reply:
Yeah agreed